defaults/main.yml

```diff
@@ -1560,8 +1718,70 @@ owncloud__apt_preferences__dependent_list_optional:
     pin: 'origin "download.owncloud.org"'
     priority: 995
     by_role: 'debops.owncloud'
-    state: '{{ owncloud__deploy_state }}'
+    state: '{{ "present"
+               if (owncloud__variant in ["owncloud"] and
+                   owncloud__deploy_state == "present")
+               else "absent" }}'
+
+
+# .. envvar:: owncloud__mariadb__dependent_databases [[[
+#
+# Configuration of the database managed by the debops.mariadb_ role.
+owncloud__mariadb__dependent_databases:
```

Comment:
Good idea with moving the database related variables to the defaults directly instead of keeping them in the playbook. :+1: